README.md

```diff
@@ -2,7 +2,12 @@
 ![Version](https://img.shields.io/github/v/release/Pennsieve/pennsieve-agent?include_prereleases)
 
 # pennsieve-agent
-Second iteration of the Pennsieve Agent
+The Pennsieve Agent is an installable application that runs on the user's local 
+machine. It manages interactions with the Pennsieve client and is a dependency
```

Comment:
@jwagenaar - essentially the agent can now be installed on a users local machine or otherwise, i.e. also runs on processors.